ProjectName/src/store/messagesSlice/index.ts

```diff
@@ -0,0 +1,12 @@
+import {useAppDispatch, useAppSelector} from '../hooks';
+import {deleteMes, MessagesSelector} from './messagesSlice';
+
+export const useMessages = () => {
+  const MessagesState = useAppSelector(MessagesSelector);
```

Comment:
Please try to stick logical chain. If you have `messagesSelector`, variable you should get from it is better name `messages`.
If you have `messagesStateSelector`, then variable would be `messagesState`.